r/R/worker_helpers.R

```diff
@@ -97,3 +97,26 @@ getClusters <- function(type, resolution, data) {
   }
   return(data)
 }
+
+handle_pagination <- function(gene_results, offset, limit, order_by, order_decreasing, filter = NULL) {
```

Comment:
Do we need to have `filter` with `NULL` optional parameter? It seems that we are always passing a `filter` param to this function